From: Andi Kleen <ak@linux.intel.com>
Date: Fri, 29 Apr 2016 17:55:48 -0700
Subject: perf/x86: Add model numbers for Kabylake CPUs
Git-commit: cba1b3798e2c4c094f2079a0d4c1ba4ec2c5a9ac
Patch-mainline: v4.6-rc7
References: FATE#320856
Signed-off-by: Tony Jones <tonyj@suse.de>

    [suse: adjust for context, no e1069839dd6893d2135b2fc4d96e5d03d73c2c3d
     perf/x86: Move perf_event_intel.c => x86/events/intel/core.c
    ]

    perf/x86: Add model numbers for Kabylake CPUs
    
    Everything the same as Skylake, just new model numbers.
    
    Signed-off-by: Andi Kleen <ak@linux.intel.com>
    Signed-off-by: Peter Zijlstra (Intel) <peterz@infradead.org>
    Link: http://lkml.kernel.org/r/1461977748-17616-1-git-send-email-andi@firstfloor.org
    Signed-off-by: Ingo Molnar <mingo@kernel.org>

---
 arch/x86/kernel/cpu/perf_event_intel.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/arch/x86/kernel/cpu/perf_event_intel.c
+++ b/arch/x86/kernel/cpu/perf_event_intel.c
@@ -3635,6 +3635,8 @@ __init int intel_pmu_init(void)
 		pr_cont("Knights Landing events, ");
 		break;
 
+	case 142: /* 14nm Kabylake Mobile */
+	case 158: /* 14nm Kabylake Desktop */
 	case 78: /* 14nm Skylake Mobile */
 	case 94: /* 14nm Skylake Desktop */
 	case 85: /* 14nm Skylake Server */
